app/models/unavailable_service.rb

```diff
@@ -0,0 +1,21 @@
+class UnavailableService
```

Comment:
I'm worried that this name might be a bit confusing relative to the rest of the codebase. As frontend service many type of pages, a developer might not understand what a "service" is in this context. It's also might be confusing as Service is used as Rails patterning for describing interactions with third parties.

```suggestion
class LocalTransactionService
```

Reply:
Also maybe using the word "Unavailable" could be confusing as we have the method unavailable too? If the class represents a unavailable service, why would we have to check if its unavailable?

Reply:
I agree - there MUST be a better name 👍

Reply:
Renamed as per suggestion